build-system/server.js

```diff
@@ -561,6 +561,22 @@ app.get('/dist/rtv/99*/*.js', function(req, res, next) {
   }).catch(next);
 });
 
+app.get('/dist/rtv/*/v0/*.js', function(req, res, next) {
```

Comment:
That means we have code adding `/dist` in local env, which is exactly what we want to remove.